### PR TITLE
New version: Fndroid.ClashForWindows version 0.20.18

### PR DESCRIPTION
**Pull request by Fndroid**
###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/99096)

### CODE REVIEW

File: manifests/f/Fndroid/ClashForWindows/0.20.18/Fndroid.ClashForWindows.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+  InstallerUrl: https://github.com/Fndroid/clash_for_windows_pkg/releases/download/0.20.18/Clash.for.Windows.Setup.0.20.18.arm64.exe
+  InstallerSha256: 3C8F5334157338B5E7B5A29033068FB3FB32CE5DC2E539839F18D26A6E0AA8EC
+  InstallerSwitches:
+    Custom: /CURRENTUSER
+- Architecture: arm64
+  Scope: user
+  InstallerUrl: https://github.com/Fndroid/clash_for_windows_pkg/releases/download/0.20.18/Clash.for.Windows.Setup.0.20.18.arm64.exe
+  InstallerSha256: 3C8F5334157338B5E7B5A29033068FB3FB32CE5DC2E539839F18D26A6E0AA8EC
+  InstallerSwitches:
+    Custom: /CURRENTUSER
```
Comment (stephengillie) [inline manifests/f/Fndroid/ClashForWindows/0.20.18/Fndroid.ClashForWindows.installer.yaml:45]:
```suggestion
```

**Comment by 499978920** [comment]
Architecture: x64 install arm64 version , error

**Comment by stephengillie** [comment]
> Architecture: x64 install arm64 version , error

This may be similar to: 
* https://github.com/microsoft/winget-cli/issues/2731
* https://github.com/microsoft/winget-pkgs/issues/95154

File: manifests/f/Fndroid/ClashForWindows/0.20.18/Fndroid.ClashForWindows.installer.yaml (hunk 1 above)
Comment (stephengillie) [inline manifests/f/Fndroid/ClashForWindows/0.20.18/Fndroid.ClashForWindows.installer.yaml:45]:
```suggestion
```

**Comment by stephengillie** [comment]
> Architecture: x64 install arm64 version , error

Thanks for pointing this out - both `x64` entries have the `arm64` InstallerUrl.

**Comment by stephengillie** [comment]
Closing as No Recent Activity.

**Pull request closed without merging**